doc/TESTS.rst.txt

```diff
@@ -66,6 +66,29 @@ or from the command line::
 
   $python runtests.py -t scipy/integrate/tests
 
+or quickly running and debugging test case in a visualize way through `vscode`_ as following.
+
+Firstly, Install the Python extension from your extensions.
+
+Second of all, open Command Palette (ctrl +shift +P) and start typing ‘python: configure tests.’,
+select pytest framework and add the following settings to your user settings.json file
+(opened with the File > Preferences > Settings)::
+
+  {
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.testing.pytestArgs": [
+        "./numpy/tests/",
+        "./numpy/core/tests/"
```

Comment:
This misses a lot of files - I think you can get away with `--pyargs numpy`

Reply:
Wonderful!  Now all of the test cases are exposed.